pkg/expr/sql/db.go

```diff
@@ -99,6 +99,7 @@ func (db *DB) writeDataframeToDb(ctx *mysql.Context, tableName string, frame *da
 	if frame == nil {
 		return fmt.Errorf("input frame is nil")
 	}
+	tableName = strings.ToLower(frame.RefID)
```

Comment:
I think you asked me yesterday why we have to lowercase the table (but I think you moved on before we got a chance to discuss this). This is new to me - I haven't imposed any requirement along these lines myself.

Reply:
I believe I saw it such that when I put in "A" (to reference RefID A), i was getting "table a not found" -- that is why I did it. If that is not the case maybe we don't need this?